crates/rerun/src/run.rs

```diff
@@ -386,6 +401,31 @@ where
     }
 }
 
+fn initialize_thead_pool(threads_args: i32) {
+    // Name the rayon threads for the benefit of debuggers and profilers:
+    let mut builder = rayon::ThreadPoolBuilder::new().thread_name(|i| format!("rayon-{i}"));
+
+    if threads_args < 0 {
+        match std::thread::available_parallelism() {
```

Comment:
I didn't know this method and I absolutely love the concept and its docs. For posterity: https://doc.rust-lang.org/std/thread/fn.available_parallelism.html